_po/ja/reference/jsonb.pot

```diff
@@ -37,7 +37,7 @@ msgid ""
 "```"
 msgstr ""
 
-msgid "You can find the JSON by full text search with `search`, `example` or `web` because all text values are full text search target."
+msgid "You can find the JSON by full text search with `server`, `example` or `web` because all text values are full text search target."
```

Comment:
```suggestion
msgid "You can find the JSON by full text search with `server`, `example`, or `web` because all text values are full text search target."
```

Reply:
fix: [d75a39e](https://github.com/pgroonga/pgroonga.github.io/pull/327/commits/d75a39e705f111c9bb72d5ad7d53f3cf551e0f15)